components/dashboard/src/admin/BlockedRepositories.tsx

```diff
@@ -113,6 +114,26 @@ export function BlockedRepositoriesList(props: Props) {
                     onClose={() => setDeleteModalVisible(false)}
                 />
             )}
+            <div
+                className={
+                    "flex rounded bg-yellow-50 dark:bg-yellow-500 text-yellow-600 dark:text-yellow-100 p-2 w-2/3 mb-2 w-full"
+                }
+            >
+                <svg
+                    className="m-1 ml-2 mr-4 bg- text-yellow-800 fill-current h-4 w-4"
+                    viewBox="0 0 16 14"
+                    fill="#FBBF24"
+                    xmlns="http://www.w3.org/2000/svg"
+                >
+                    <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M6.25706 1.09882C7.02167 -0.26048 8.97875 -0.260481 9.74336 1.09882L15.3237 11.0194C16.0736 12.3526 15.1102 13.9999 13.5805 13.9999H2.4199C0.890251 13.9999 -0.0731769 12.3526 0.676753 11.0194L6.25706 1.09882ZM9.00012 11C9.00012 11.5523 8.55241 12 8.00012 12C7.44784 12 7.00012 11.5523 7.00012 11C7.00012 10.4477 7.44784 10 8.00012 10C8.55241 10 9.00012 10.4477 9.00012 11ZM8.00012 3C7.44784 3 7.00012 3.44772 7.00012 4V7C7.00012 7.55228 7.44784 8 8.00012 8C8.55241 8 9.00012 7.55228 9.00012 7V4C9.00012 3.44772 8.55241 3 8.00012 3Z"
+                        fill="currentColor"
+                    />
+                </svg>
+                <span>Entries in this table have an immediate effect on all users. Please use it carefully.</span>
```

Comment:
**suggestion**: I'd put that warning alert component inside the modal when adding a new block repository to keep ti close to the moment of action.